admin/js/monk-admin.js

```diff
@@ -66,6 +66,49 @@
 			return false;
 		});
 
+		var not_submit = true;
+		$( document ).on( 'submit', '#monk-form-settings', function( event ) {
+			if ( not_submit ) {
+				event.preventDefault();
+				not_submit = false;
+				$( '#monk-spinner' ).addClass( 'is-active' );
+				var form_data = $( '#monk-form-settings' ).serializeArray();
+
+				form_data.push({
+					name : 'action',
+					value : 'monk_save_language_packages',
+				});
+				$.ajax({
+					type: 'POST',
+					url: monk.ajax_url,
+					data: form_data,
+					success: function( response ) {
```

Comment:
On sucess, remove all fake notices.
Let's just add a message next to the submit button spinner updating the status.
 - Downloading packages
 - Defining language for posts and terms
 - Done!